templates/Procfile

```diff
@@ -1,2 +0,0 @@
-web: bundle exec puma -p $PORT -C ./config/puma.rb
-worker: bundle exec rake jobs:work
```

Comment:
Kinda. We use `rails server -p $PORT`. https://github.com/heroku/heroku-buildpack-ruby/blob/52b7238c748678fffc70c8f2a51e795305ef15c9/lib/language_pack/rails3.rb#L27